graphs/possible_bipartition.py

```diff
@@ -1,12 +1,72 @@
 # Can be used for BFS
-from collections import deque 
+from collections import deque
 
 def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n^2)
+        Space Complexity: O(n)
     """
```

Comment:
I would say this is O(NE) since you're essentially doing a BFS visiting every node and traveling along each edge at least once.